livefs_edit/context.py

```diff
@@ -183,6 +183,11 @@ def get_arch(self):
         with open(self.p('new/iso/.disk/info')) as fp:
             return fp.read().strip().split()[-2]
 
+    def get_suite(self):
+        path=glob.glob('dists/*/Release')
```

Comment:
Doesn't this need to be `ctxt.p('old/iso/dists/*/Release')` or something like that?

Reply:
Good catch. I fixed it in [c232f19](https://github.com/mwhudson/livefs-editor/pull/19/commits/c232f195f703da60557178e9060eb26064c36210)